.github/ISSUE_TEMPLATE/rfc.yml

```diff
@@ -0,0 +1,49 @@
+name: RFC
+description: Start gathering information regarding new functionality
+title: "[RFC]: "
+labels: [RFC]
+assignees:
+  - andrzejewski
+  - krzysztofzuraw
```

Comment:
**question**: should this be predefined, do we assign whole team or just you two for all future RFCs?

Reply:
@Droniu It's about moderation, but who does this RFC, we just need to be notified